modules/Campaigns/WizardNewsletter.php

```diff
@@ -393,20 +393,31 @@
 
 // ----- show target lists...
 
+if (!isset($targetListDataArray)) {
+    $targetListDataArray = array();
+}
+if (!isset($targetListDataAssoc)) {
+    $targetListDataAssoc = array();
+}
+
 $targetList = BeanFactory::getBean('ProspectLists')->get_full_list();
 
-$targetListData = array();
-foreach($targetList as $prospectLst) {
-    $nxt = array(
-        'id' => $prospectLst->id,
-        'name' => $prospectLst->name,
-        //'type' => $prospectLst->type,
-        'description' => $prospectLst->description,
-        'type' => $prospectLst->list_type,
-        'count' => $prospectLst->get_entry_count(),
-    );
-    $targetListDataArray[] = $nxt;
-    $targetListDataAssoc[$prospectLst->id] = $nxt;
+if($targetList) {
+    $targetListData = array();
+    foreach ($targetList as $prospectLst) {
+        $nxt = array(
```

Comment:
I think "nxt" means "next" of the results, look the next line. btw it's only a guess, in my PR I just changed the indent of these line.